saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
 msgid "No results found"
 msgstr "Aucun résultat trouvé"
 
+#: templates/training/training_request.html:8
+#: templates/training/view_request.html:8
+msgid "Training Form Request"
+msgstr "Demande de formulaire de formation"
+
+#: templates/training/training_request.html:11
+#: templates/training/view_request.html:11
+msgid "Training Request Form"
+msgstr "Demande de formulaire de formation"
+
+#: templates/training/training_request.html:12
+msgid "Please fill out this form to apply for training."
+msgstr "Veuillez remplir ce formulaire pour postuler à une formation."
```

Comment:
To be confirmed: postuler probably needs to be modified by another verb as this is mostly associated to applying to a job, in my experience.

Reply:
Who do you think is the best person/resource to confirm with? Translations?

And BTW, thank you so much for going over the translations!